SmartDeviceLinkTests/ProxySpecs/SDLSecondaryTransportManagerSpec.m

```diff
@@ -1011,12 +1002,7 @@ __block void (^sendNotificationForHMILevel)(SDLHMILevel hmiLevel) = ^(SDLHMILeve
                     [manager.stateMachine setToState:SDLSecondaryTransportStateReconnecting fromOldState:nil callEnterTransition:YES];
                 });
 
-                // wait for the timer
-                float waitTime = RetryConnectionDelay;
-                NSLog(@"Please wait for reconnection timeout... (for %.02f seconds)", waitTime);
-                [NSThread sleepForTimeInterval:waitTime];
-
-                expect(manager.stateMachine.currentState).toEventually(equal(SDLSecondaryTransportStateConfigured));
+                expect(manager.stateMachine.currentState).withTimeout(RegisterTransportTime + 3.0).toEventually(equal(SDLSecondaryTransportStateConfigured));
```

Comment:
Did you mean to use `RetryConnectionDelay` instead of `RegisterTransportTime`?

Reply:
🤦